pkg/controller/privatelink/actuator/gcpactuator/gcplinkactuator.go

```diff
@@ -145,45 +149,61 @@ func (a *GCPLinkActuator) Reconcile(cd *hivev1.ClusterDeployment, metadata *hive
 		return reconcile.Result{}, errors.Wrap(err, "failed to find the cluster api Forwarding Rule")
 	}
 
-	logger.Debug("reconciling Service Attachment Subnet")
-	subnetModified, subnet, err := a.ensureServiceAttachmentSubnet(cd, metadata, network)
-	if err != nil {
-		logger.WithError(err).Error("failed to reconcile the Service Attachment Subnet")
-
-		err := conditions.SetErrConditionWithRetry(*a.client, cd, "ServiceAttachmentSubnetReconcileFailed", errors.New(controllerutils.ErrorScrub(err)), logger)
+	var subnet *compute.Subnetwork
+	// Use a preexisting subnet if configured.
+	if subnetName := getServiceAttachmentSubnetExistingName(cd); subnetName != "" {
+		subnet, err = a.gcpClientSpoke.GetSubnet(subnetName, cd.Spec.Platform.GCP.Region, getServiceAttachmentSubnetExistingProject(cd))
```

Comment:
nit: debug log saying we're using existing subnet?

Reply:
It exists a few lines lower (159) using values from the retrieved subnet. Though, that value should be equal to the subnetName and could be moved higher up. Would you prefer that?

Reply:
oh, no, I just missed it, it's cool.